proxy/plugin/add_header.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+"""
+    proxy.py
+    ~~~~~~~~
+    ⚡⚡⚡ Fast, Lightweight, Pluggable, TLS interception capable proxy server focused on
+    Network monitoring, controls & Application development, testing, debugging.
+
+    :copyright: (c) 2013-present by Abhinav Singh and contributors.
+    :license: BSD, see LICENSE for more details.
+"""
+import re
+from typing import Optional
+
+from ..common.flag import flags
+from ..http.parser import HttpParser
+from ..http.proxy import HttpProxyBasePlugin
+
+DEFAULT_HEADER = "X-Proxy-Py:Proxied with proxy.py"
```

Comment:
I think we don't need any default headers from a plugin.  This will be little surprising for end user who has no idea about default headers added by the plugin.  Wdyt?